samples/client/petstore/csharp/httpclient/net9/Petstore/src/Org.OpenAPITools/Model/EntityBase.cs

```diff
@@ -62,7 +62,7 @@ protected EntityBase()
         /// <summary>
         /// Gets or Sets Schema
         /// </summary>
-        [DataMember(Name = "$schema", IsRequired = true, EmitDefaultValue = true)]
+        [DataMember(Name = "$schema", IsRequired = true, EmitDefaultValue = false)]
```

Comment:
A required property was giveen a default value which does not align with expectation